### PR TITLE
Add test for ana4Prob probAna

### PR DESCRIPTION
**Pull request by awirb**
* remove the use of np.matrix in file writing 
* adjust result asc file writing in ascUtils and use it in probAna

### CODE REVIEW

File: avaframe/ana4Prob/probAna.py [hunk 1]
```diff
@@ -59,6 +59,8 @@ def probAnalysis(avaDir, cfg, cfgMain, inputDir='', outDir=''):
             data = np.loadtxt(fileName, skiprows=6)
             dataLim = np.zeros((nRows, nCols))
 
+            log.info('File name is %s' % fileName)
```
Comment (fso42) [inline avaframe/ana4Prob/probAna.py:62]: maybe debug level?

**Review by fso42 — APPROVED** [review]

File: avaframe/in2Trans/ascUtils.py [hunk 2]
```diff
@@ -142,34 +142,35 @@ def readRaster(fname):
...
+    #                             filetypes=[('ASCII', '*.asc')],
+    #                             initialfile='path.asc',
+    #                             initialdir=os.getcwd())
+    # else:
+    #     outfile = asksaveasfile(mode='w',
+    #                             defaultextension='.asc',
+    #                             title='Save Output File',
+    #                             filetypes=[('ASCII', '*.asc')],
+    #                             initialfile='path.asc',
+    #                             initialdir=os.getcwd())
```
Comment (matthiasto) [inline avaframe/in2Trans/ascUtils.py:161]: you can probably remove this

**Review by matthiasto — APPROVED** [review]

File: avaframe/ana4Prob/probAna.py (hunk 1 above)
Comment (awirb) [reply avaframe/ana4Prob/probAna.py:62]: fine, changed this

File: avaframe/in2Trans/ascUtils.py (hunk 2 above)
Comment (awirb) [reply avaframe/in2Trans/ascUtils.py]: done

**Pull request merged**